src/api/getContext.ts

```diff
@@ -97,9 +98,14 @@ export default async function getContext (opts: StrictSupiOptions, installType?:
     existsCurrentShrinkwrap: !!files[2],
     storeController: files[3],
     skipped: new Set(modules && modules.skipped || []),
+    pendingBuilds: [],
```

Comment:
you can assign the value here. `pendingBuilds: modules && modules.pendingBuilds || []`